django/views/debug.py

```diff
@@ -893,6 +893,28 @@ def default_urlconf(request):
   <h2>Request information</h2>
 
 {% if request %}
+  <h3 id="user-info">USER</h3>
+  {% if request.user %}
+    <table class="req">
+      <thead>
+        <tr>
+          <th>Variable</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for var in request.user.items %}
```

Comment:
:grimacing: My bad, I ran the selenium view_tests/ but did not see that exception.  What would be a better way to test?

Does the below look more in line with what you were envisioning? 
 ```
<h3 id="user-info">USER</h3>
  {% if request.user %}
    <p>{{ request.user }}</p>
  {% else %}
    <p>No USER data</p>
  {% endif %}
```